docs/config/omniauth.md

```diff
@@ -21,13 +22,14 @@ In `config/initializers/omniauth.rb`, add the settings for each of your provider
 
 These settings must be obtained from the providers themselves.
 
-**Example using github, facebook, and google**:
+**Example using Github, Facebook, Google, and Apple**:
 ~~~ruby
 # config/initializers/omniauth.rb
 Rails.application.config.middleware.use OmniAuth::Builder do
   provider :github,        ENV['GITHUB_KEY'],   ENV['GITHUB_SECRET'],   scope: 'email,profile'
   provider :facebook,      ENV['FACEBOOK_KEY'], ENV['FACEBOOK_SECRET']
   provider :google_oauth2, ENV['GOOGLE_KEY'],   ENV['GOOGLE_SECRET']
+  provider :apple,         ENV['APPLE_CLIENT_ID'], '', { scope: 'email name', team_id: ENV['APPLE_TEAM_ID'], key_id: ENV['APPLE_KEY'], pem: ENV['APPLE_PEM'] }
```

Comment:
This provider does not accept a secret directly, rather it uses a p8 PEM certificate to generate JWT authorization tokens on the fly for provider handshake.